django/forms/widgets.py

```diff
@@ -875,10 +875,10 @@ class SplitDateTimeWidget(MultiWidget):
     supports_microseconds = False
     template_name = 'django/forms/widgets/splitdatetime.html'
 
-    def __init__(self, attrs=None, date_format=None, time_format=None):
+    def __init__(self, attrs=None, date_format=None, time_format=None, date_attrs=None, time_attrs=None):
```

Comment:
@pope1ni Thank for review.

IMO `date_attrs` and `time_attrs` will be used only in some special cases when you want to have different subwidgets attributes (at least some of them, like `class`). `attrs` always override `date_attrs`, `time_attrs` in `get_context()` just like in `MultiWidget` (see [widgets.py#L800](https://github.com/django/django/blob/master/django/forms/widgets.py#L800)). I think that
```
SplitDateTimeWidget(attrs=attrs, date_attrs=date_attrs, time_attrs=time_attrs)
```
should work the same as
```
MultiWidget(widgets=(DateInput(date_attrs), TimeInput(time_attrs)), attrs=attrs)
```
I updated PR.